CMakeLists.txt

```diff
@@ -113,8 +113,33 @@ install(
     EXPORT ${PROJECT_NAME}-targets)
 
 if(UMF_BUILD_OS_MEMORY_PROVIDER AND (LINUX OR WINDOWS))
-    # Do not use pkgconfig, because it sets wrong paths on Windows
-    find_package(LIBHWLOC REQUIRED hwloc)
+    pkg_check_modules(LIBHWLOC hwloc)
+    if(NOT LIBHWLOC_FOUND)
+        find_package(LIBHWLOC REQUIRED hwloc)
+    endif()
+    message(STATUS ">>>>> LIBHWLOC_LIBRARIES = ${LIBHWLOC_LIBRARIES}")
+    message(STATUS ">>>>> LIBHWLOC_LINK_LIBRARIES = ${LIBHWLOC_LINK_LIBRARIES}")
+    message(STATUS ">>>>> LIBHWLOC_LIBRARY_DIRS = ${LIBHWLOC_LIBRARY_DIRS}")
+    message(STATUS ">>>>> LIBHWLOC_LDFLAGS = ${LIBHWLOC_LDFLAGS}")
+    message(STATUS ">>>>> LIBHWLOC_LDFLAGS_OTHER = ${LIBHWLOC_LDFLAGS_OTHER}")
+    message(STATUS ">>>>> LIBHWLOC_INCLUDE_DIRS = ${LIBHWLOC_INCLUDE_DIRS}")
+    message(STATUS ">>>>> LIBHWLOC_CFLAGS = ${LIBHWLOC_CFLAGS}")
+    message(STATUS ">>>>> LIBHWLOC_CFLAGS_OTHER = ${LIBHWLOC_CFLAGS_OTHER}")
+endif()
+
+if(UMF_BUILD_LIBUMF_POOL_SCALABLE AND (LINUX OR WINDOWS))
+    pkg_check_modules(TBB tbb)
+    if(NOT TBB_FOUND)
+        find_package(TBB REQUIRED tbb)
+    endif()
+    message(STATUS ">>>>> TBB_LIBRARIES = ${TBB_LIBRARIES}")
+    message(STATUS ">>>>> TBB_LINK_LIBRARIES = ${TBB_LINK_LIBRARIES}")
+    message(STATUS ">>>>> TBB_LIBRARY_DIRS = ${TBB_LIBRARY_DIRS}")
+    message(STATUS ">>>>> TBB_LDFLAGS = ${TBB_LDFLAGS}")
+    message(STATUS ">>>>> TBB_LDFLAGS_OTHER = ${TBB_LDFLAGS_OTHER}")
+    message(STATUS ">>>>> TBB_INCLUDE_DIRS = ${TBB_INCLUDE_DIRS}")
+    message(STATUS ">>>>> TBB_CFLAGS = ${TBB_CFLAGS}")
+    message(STATUS ">>>>> TBB_CFLAGS_OTHER = ${TBB_CFLAGS_OTHER}")
```

Comment:
Is it only for debugging purposes? I'd prefer to build it with a verbose param or something like that instead of leaving it here.